client/src/components/Organizer/JudgeInfo/JudgeInfo.jsx

```diff
@@ -109,9 +111,10 @@ class JudgeInfo extends Component {
             [prevState.curr_name, prevState.selected.label]
           ])
           return {
-          info: newInfo,
-          curr_name: '',
-          selected: ''
+            reassignJudges: true, 
+            info: newInfo,
+            curr_name: '',
```

Comment:
you didnt lint did you.......

Reply:
I ran lint and nothing errored on this line?

Reply:
currName not curr_name

Reply:
oh curr_name has been there since this file was first written... ill fix it tho